tox.ini

```diff
@@ -90,3 +90,76 @@ commands =
     nosetests --rednose --immediate -sv contrib/runners/orquesta_runner/tests/integration/
     nosetests --rednose --immediate -sv st2tests/integration/orquesta/
     nosetests --rednose --immediate -sv contrib/runners/python_runner/tests/integration/
+
+# Python 3.8 tasks
```

Comment:
IIRC, we don't use tox for anything anymore. 

If that's indeed the case, we should probably just remove tox.ini to reduce confusion and amount of code we need to maintain.

IIRC, we used it in the past when we still supported Python 2 for running tests under Python 3.


What do other ppl think?

Reply:
I just pushed a change which removes it - d345d6bb6.

If we ever plan to utilize it again in the future (e.g. if / when we switch to pytest), we can add it back then. Until that there is point for a code which is not being utilized to be left in the repo :)